src/main/java/carpet/logging/Logger.java

```diff
@@ -219,9 +215,13 @@ public void onPlayerConnect(Player player, boolean firstTime)
         else if(firstTime)
         {
             Set<String> loggingOptions = new HashSet<>(Arrays.asList(CarpetSettings.defaultLoggers.split(",")));
-            if (loggingOptions.contains(getLogName()))
-            {
-                LoggerRegistry.subscribePlayer(playerName, getLogName(), getDefault());
+            String logName = getLogName();
+            for (String str : loggingOptions) {
+                String[] vars = str.split(" ");
+                if (vars[0].equals(logName)) {
+                    LoggerRegistry.subscribePlayer(playerName, getLogName(), vars.length == 1 ? getDefault() : vars[1]);
```

Comment:
It's probably not supported/not a thing anywhere, but this still wouldn't work if a logger has parameters with spaces, as only the first parameter would be passed (`vars[1]`). I think it can be solved with:
```suggestion
                String[] vars = str.split(" ", 2);
                if (vars[0].equals(logName)) {
                    LoggerRegistry.subscribePlayer(playerName, getLogName(), vars.length == 1 ? getDefault() : vars[1]);
```